.github/workflows/ci.yml

```diff
@@ -3,7 +3,31 @@ name: Run unit tests & check conventions
 on: [push, pull_request]
 
 jobs:
-  tests:
+  file-conventions:
+    runs-on: ubuntu-22.04
+    container:
+      image: "ubuntu:22.04"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install required dependencies
+        run: |
+          apt update
+          apt install -y sudo
+          sudo apt install -y curl
+          sudo apt-get install -y libicu-dev
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 6.0.x
+      - name: compile the FileConventions project
+        run: |
+          dotnet build FileConventions.Test/FileConventions.Test.fsproj
+          dotnet build FileConventions/FileConventions.fsproj
```

Comment:
AFAIU FileConventions.Test depends on FileConventions, so either you change the order of these two lines, or just leave the first one because the first one is already building both projects